### PR TITLE
Test with main on Lomap dev branch

### LINKED ISSUE #856: Fix OpenFE tests to match main on Lomap
The openfe tests depend on some of the old LomapAtomMapper defaults (especially a large max3d and shift=True).

This is because some of the basic test files are unalligned molecules.

To ensure the tests still pass we need to pin these defaults in some of our tests.

### PR DESCRIPTION
**Pull request by IAlibay**
Fixes #856 

Checklist
* [x] Added a ``news`` entry

## Developers certificate of origin
- [x] I certify that this contribution is covered by the MIT License [here](https://github.com/OpenFreeEnergy/openfe/blob/main/LICENSE) and the **Developer Certificate of Origin** at <https://developercertificate.org/>.

### CODE REVIEW

File: environment.yml [hunk 1]
```diff
@@ -44,4 +44,5 @@ dependencies:
   - pip:
     - sphinx-toolbox
     - openff-nagl-models>=0.1.2
+    - git+https://github.com/OpenFreeEnergy/Lomap@main
```
Comment (IAlibay) [inline environment.yml:47]: remember to  change back!

File: environment.yml [hunk 2]
```diff
@@ -9,6 +9,7 @@ dependencies:
   - numpy
   - networkx
   - rdkit
+  - packaging
```
Comment (IAlibay) [inline environment.yml:12]: I think this is the healthiest way of doing (at least MDA just puts a direct dependency on packaging)

File: environment.yml (hunk 1 above)
Comment (IAlibay) [reply environment.yml:47]: Last batch of tests all went green so removing this now!

File: environment.yml (hunk 2 above)
Comment (mikemhenry) [reply environment.yml:12]: Sounds good!

**Review by mikemhenry — APPROVED** [review]
LGTM! So we have a follow up todo, re-write these tests so we can use the current defaults of lomap (or the defaults we suggest)

**Pull request merged**